crates/starknet-server/tests/get_transaction_receipt_by_hash.rs

```diff
@@ -107,7 +107,7 @@ mod get_transaction_receipt_by_hash_integration_tests {
         // declare the contract
         let declaration_result = predeployed_account
             .declare(Arc::new(cairo_1_contract), casm_class_hash)
-            .max_fee(FieldElement::from(1e18 as u128))
+            .max_fee(FieldElement::from(10000000000000000000u128))
```

Comment:
Why there is a change from "1e18 as u128" to "10000000000000000000u128"? I liked the previous version.

Reply:
because 1e18 is not the right amount that should be used

Reply:
but can't we just use 1e19?

Reply:
https://github.com/0xSpaceShard/starknet-devnet-rs/commit/b6e50e99a8619be3420a3b98ad3f9c09d239807f